test/test/cpu/CpuTests.java

```diff
@@ -14,7 +14,7 @@
 public class CpuTests {
 
     private static void assertCloseTo(long value, long target, String message) {
-        Assert.isGreaterOrEqual(value, target * 0.75, message);
+        Assert.isGreaterOrEqual(value, target * 0.6, message);
```

Comment:
40% difference is a lot - it may absorb some real issues.
This is a severe change, I'm reluctant to accept this for all cpu tests on all platforms, especially since we haven't seen problems with this test on GitHub Actions or on EC2 instances.

Reply:
Ok, I'll try to reduce it.

Reply:
I'm now computing the cpu/wall ratio and use it to get a better expected CPU sample time. Additionally, I skip the test if the cpu/wall ratio is lower than 0.5, to prevent failing on heavily loaded systems.